src/Json.Schema.ToDotNet.UnitTests/DataModelGeneratorTests.cs

```diff
@@ -2363,7 +2369,152 @@ public int GetHashCode(C obj)
+    {
+        /// <summary>
+        /// An integer property.
+        /// </summary>
+        [DataMember(Name = ""intProp"", IsRequired = false, EmitDefaultValue = false)]
+        public int IntProp { get; set; }
+
+        /// <summary>
+        /// An integer property with a default value.
+        /// </summary>
+        [DataMember(Name = ""intPropWithDefault"", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue(42)]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        public int IntPropWithDefault { get; set; }
+
+        /// <summary>
+        /// A number property.
+        /// </summary>
+        [DataMember(Name = ""numberProp"", IsRequired = false, EmitDefaultValue = false)]
+        public double NumberProp { get; set; }
+
+        /// <summary>
+        /// A number property with a default value.
+        /// </summary>
+        [DataMember(Name = ""numberPropWithDefault"", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue(42.1)]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        public double NumberPropWithDefault { get; set; }
+
+        /// <summary>
+        /// A string property.
+        /// </summary>
+        [DataMember(Name = ""stringProp"", IsRequired = false, EmitDefaultValue = false)]
+        public string StringProp { get; set; }
+
+        /// <summary>
+        /// A string property with a default value.
+        /// </summary>
+        [DataMember(Name = ""stringPropWithDefault"", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue(""42"")]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        public string StringPropWithDefault { get; set; }
+
+        /// <summary>
+        /// A Boolean property.
+        /// </summary>
+        [DataMember(Name = ""boolProp"", IsRequired = false, EmitDefaultValue = false)]
+        public bool BoolProp { get; set; }
+
+        /// <summary>
+        /// A Boolean property with a true default value.
+        /// </summary>
+        [DataMember(Name = ""boolPropWithTrueDefault"", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue(true)]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        public bool BoolPropWithTrueDefault { get; set; }
+
+        /// <summary>
+        /// A Boolean property with a false default value.
+        /// </summary>
+        [DataMember(Name = ""boolPropWithFalseDefault"", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue(false)]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        public bool BoolPropWithFalseDefault { get; set; }
+    }
+}";
+
+            var generator = new DataModelGenerator(_settings, _testFileSystem.FileSystem);
+            JsonSchema schema = SchemaReader.ReadSchema(Schema, TestUtil.TestFilePath);
```

Comment:
>TestUtil [](start = 64, length = 8)

'TestUtil', boo. are we programming in the 80's? you can take the engineers out of the eighties i suppose, but you can't completely take the eighties from the engineer. cf. earlier references to 'hitchhiker's guide to the galaxy'. actually published in 10/79, of course, but read by us, no doubt, in the subsequent decade

Reply:
Ugh. I'm not changing the 53 occurrences of this name. We can laugh at our younger selves, but I don't see this name as a barrier to readability. It's just a bit old-fashioned. That's why Terisa calls me "Fifties Man".

---
In reply to: [244508719](https://github.com/Microsoft/jschema/pull/62#discussion_r244508719) [](ancestors = 244508719)

Reply:
Born in the Fifties. Formed by the Eighties. Perhaps you enjoyed the Police song 'Born in the Fifties' at that time.

---
In reply to: [244512315](https://github.com/Microsoft/jschema/pull/62#discussion_r244512315) [](ancestors = 244512315,244508719)